content/zh/devops/_index.md

```diff
@@ -5,47 +5,47 @@ layout: "scenario"
-  content: KubeSphere DevOps provides CI/CD pipeline based on Jenkins, and offers automated workflows including binary-to-image (B2I) and source-to-image (S2I), helps organizations accelerate time to market for their product.
+  title: KubeSphere DevOps 提供端到端的工作流，集成主流 CI/CD 工具，提升交付能力
+  content: KubeSphere DevOps 提供基于 Jenkins 的 CI/CD 流水线，并提供自动化工作流，包括 Binary-to-Image (B2I) 和 Source-to-Image (S2I)，帮助不同的组织加快产品上市时间。
   image: /images/devops/banner.jpg
 
 image: /images/devops/dev-ops.png
 
 section2:
-  title: Automatically Checkout Code, Test, Analyse, Build, Deploy and Release
+  title: 自动检出 (Checkout) 代码、测试、分析、构建、部署并发布
   list:
-    - title: Out-of-box CI/CD Pipeline
+    - title: 开箱即用的 CI/CD 流水线
       image: /images/devops/CD-pipeline.png
       contentList:
-        - content: <span>Easy to integrate with your SCM,</span> supporting GitLab / GitHub / BitBucket / SVN
-        - content: <span>Design a graphical editing panel</span> to create CI/CD pipelines, without writing Jenkinsfile
-        - content: <span>Integrate SonarQube</span> to implement source code quality analysis
-        - content: <span>Support dependency cache</span> to accelerate build and deployment
-        - content: <span>Provide dynamic build agents</span> to automatically spin up Pods as necessary
+        - content: <span>易于集成至您的 SCM，</span>支持 GitLab/GitHub/BitBucket/SVN
+        - content: <span>图形编辑面板设计，</span>可创建 CI/CD 流水线且无需编写 Jenkinsfile
+        - content: <span>集成 SonarQube，</span>实现源代码质量分析
+        - content: <span>支持依赖项缓存，</span>加快构建和部署
+        - content: <span>提供动态构建 Agent，</span>根据需要自动创建 Pod
 
-    - title: Built-in Automated Toolkits
+    - title: 内置自动化工具箱
       image: /images/devops/Built-in-automated-toolkits.png
       contentList:
-        - content: <span>Source to Image</span> builds reproducible container images from source code without writing dockerfile
-        - content: <span>Binary-to-image</span> is the bridge between your artifact and a runnable image
-        - content: <span>Support automatically building and pushing</span> images to any registry, and finally deploy them to Kubernetes
-        - content: <span>Provide excellent recoverability and flexibility</span> as you can rebuild and rerun S2I / B2I whenever a patch is needed
+        - content: <span>Source to Image</span> 从源代码构建可再现容器镜像，无需编写 Dockerfile
+        - content: <span>Binary-to-image</span> 将您的制品自动构建成可运行镜像
+        - content: <span>支持自动化构建和推送</span>镜像至任意仓库，并最终部署至 Kubernetes
+        - content: <span>提供卓越的可恢复性和灵活性，</span>您可以在需要补丁时重新构建并重新运行 S2I/B2I
 
-    - title: Use GitOps to implement DevOps, not just culture
+    - title: 使用 GitOps 实现 DevOps（包括其文化）
       image: /images/devops/Clear-insight.png
       contentList:
-        - content: <span>Combine Git with Kubernetes’ convergence, and automates the cloud native Apps delivery</span>
-        - content: <span>Designed for teams, offer built-in multitenancy in DevOps project</span>
```

Comment:
Change the English source text to "Designed for DevOps teamwork on the basis of the multi-tenant system of KubeSphere"